lib/matplotlib/scale.py

```diff
@@ -544,39 +544,35 @@ class SymmetricalLogScale(ScaleBase):
     need to have a range around zero that is linear.  The parameter
     *linthresh* allows the user to specify the size of this range
     (-*linthresh*, *linthresh*).
+
+    Parameters
```

Comment:
Not to hold up this PR but we may want to record somewhere the decision of whether constructor parameters should be documented in the class' docstring or in the `__init__` docstring.

Reply:
I'm going off the `numpydoc` guide (https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring), but I agree it would be nice to document this in our own doc guide.